src/sentry/integrations/github/client.py

```diff
@@ -97,20 +97,20 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> List[str]:
             if msg == "Git Repository is empty.":
                 logger.warning(f"{repo_full_name} is empty.")
             elif msg == "Not Found":
-                logger.error(f"The Github App does not have access to {repo_full_name}.")
```

Comment:
Calling `logger.error` actually creates Sentry issues. I'm hoping `logger.warning` does not.

Reply:
I think that's correct

Reply:
A [sample issue](https://sentry.io/organizations/sentry/issues/3727674865/?project=1&query=is%3Aunresolved+derive+level%3Aerror&referrer=issue-stream&statsPeriod=14d) for the curious:
<img width="290" alt="image" src="https://user-images.githubusercontent.com/44410/201103252-5e60bb83-a80f-450e-b488-e8c63712eacc.png">